docs/input/docs/reference/configuration.md

```diff
@@ -390,19 +412,19 @@ Take this commit graph
 
 By looking at this graph, you cannot tell which of these scenarios happened:
 
-*   feature/foo branches off release/1.0.0
-    *   Branch release/1.0.0 from main
-    *   Branch feature/foo from release/1.0.0
-    *   Add a commit to both release/1.0.0 and feature/foo
-    *   release/1.0.0 is the base for feature/foo
-*   release/1.0.0 branches off feature/foo
-    *   Branch feature/foo from main
-    *   Branch release/1.0.0 from feature/foo
-    *   Add a commit to both release/1.0.0 and feature/foo
-    *   feature/foo is the base for release/1.0.0
++ feature/foo branches off release/v1.0.0
+  + Branch release/v1.0.0 from main
+  + Branch feature/foo from release/v1.0.0
+  + Add a commit to both release/v1.0.0 and feature/foo
+  + release/v1.0.0 is the base for feature/foo
++ release/v1.0.0 branches off feature/foo
+  + Branch feature/foo from main
+  + Branch release/v1.0.0 from feature/foo
+  + Add a commit to both release/v1.0.0 and feature/foo
+  + feature/foo is the base for release/v1.0.0
```

Comment:
Why the change from `*` to `+`?

Reply:
This wasn't an intended change; it was because I started my changes before commit 7809b809da2268e1d11ea55bf710d727674ef2d9. I will revert.